lua/lazyvim/util/toggle.lua

```diff
@@ -41,10 +41,10 @@ function M.number()
   end
 end
 
-local enabled = true
+local status = vim.diagnostic.is_disabled()
 function M.diagnostics()
-  enabled = not enabled
-  if enabled then
+  status = not status
+  if not status then
```

Comment:
It might be clearer if you just change to:
```lua
local enabled = not vim.diagnostic.is_disabled()
```
and then I think the rest of the code can stay the same?

Super minor either way, please do what you think is best.

Reply:
Thank you @jyuan0. 

Always keep thinking it in my mind as status, but no need to make unnecessary changes to existing terminology.